src/serde_deser.rs

```diff
@@ -0,0 +1,820 @@
+use std::collections::hash_map::IntoIter;
```

Comment:
Maybe we should add something like this? Could be helpful in the future.

```suggestion
//! Deserialize into any serde type
//!
//! Sourced from <https://github.com/serde-rs/json/blob/master/src/value/de.rs>

use std::collections::hash_map::IntoIter;
```